test/distributed/org/apache/cassandra/distributed/upgrade/DropCompactStorageNullClusteringValuesTest.java

```diff
@@ -33,7 +33,7 @@ public class DropCompactStorageNullClusteringValuesTest extends UpgradeTestBase
     public void testNullClusteringValues() throws Throwable
     {
         new TestCase().nodes(1)
-                      .upgradesToCurrentFrom(v30)
+                      .upgradesToCurrentFrom(OLDEST)
                       .withConfig(config -> config.with(GOSSIP, NETWORK, NATIVE_PROTOCOL).set("enable_drop_compact_storage", true))
                       .setup(cluster -> {
                           String create = "CREATE TABLE %s.%s(k int, c1 int, c2 int, v int, PRIMARY KEY (k, c1, c2)) " +
```

Comment:
is it still possible to `CREATE TABLE WITH COMPACT STORAGE` in 4.0+  ? 

ref: https://github.com/apache/cassandra/pull/3782#issuecomment-2578362286

Reply:
It's still possible